lib/pxcontext.js

```diff
@@ -52,6 +52,11 @@ class PxContext {
                 }
             });
         }
+        if (this.uri.includes('graphql')) {
+            config.logger.debug('Graphql route detected');
+            this.graphqlData = pxUtil.getGraphqlData(req);
```

Comment:
Why are you keeping it as a member?

Reply:
So that we can report the info in the risk_api call